.github/workflows/codeql-analysis.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         setup-python-dependencies: false
 
     - name: Install remotecv
-      run: make setup
+      run: sudo make setup
```

Comment:
@devppjr if you install the python packages with `sudo`, then the Python environment wouldn't be cached anymore.
We either need to change the cache path or remove the `sudo`.

Reply:
@guilhermef I did these changes to fix the ci. I tried a lot of changes, but just this changes fixed the ci. Could you help me with the ci issue?

Reply:
fixed in https://github.com/thumbor/remotecv/pull/64. waiting merge!